Update the weston docker compose

AllanKamimura opened the pull request:
- Add a command to start some weston apps

Closes: #16

AllanKamimura committed:
Update the weston docker compose

- Add a command to start some weston apps

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

leograba commented:
@AllanKamimura did you consider starting this demo with the [--developer](https://developer.toradex.com/torizon/application-development/working-with-containers/working-with-weston-on-torizoncore/#configuration-options) flag so people can access the top bar menu? That makes it interactive so the user could open a terminal, etc.

That's just an idea of course.

AllanKamimura committed:
weston: Add the --developer flag

This allows the user to use the top navigation bar,
which includes a terminal and a clock.

This also gives a better background image

Signed-off-by: Allan Henrique Kamimura <allan.kamimura@toradex.com>

AllanKamimura commented:
with the developer flag, the demo should look like this

![screenshot](https://github.com/user-attachments/assets/af083c96-2746-4292-9924-6ade7e7c374b)
